tests/integration/azure/test_azure_credentials.py

```diff
@@ -0,0 +1,142 @@
+import base64
+import re
+from unittest.mock import MagicMock
+
+import pytest
+from databricks.labs.blueprint.tui import MockPrompts
+
+from databricks.labs.ucx.assessment.azure import AzureServicePrincipalInfo
+from databricks.labs.ucx.assessment.crawlers import _SECRET_PATTERN
+from databricks.labs.ucx.azure.access import StoragePermissionMapping
+from databricks.labs.ucx.azure.azure_credentials import AzureServicePrincipalMigration
+
+
+@pytest.fixture
+def prepare_spn_migration_test(ws, debug_env, make_random):
+    def inner(read_only=False) -> dict:
+        spark_conf = ws.clusters.get(cluster_id=debug_env["TEST_LEGACY_SPN_CLUSTER_ID"]).spark_conf
```

Comment:
```suggestion
        spark_conf = ws.clusters.get(debug_env["TEST_LEGACY_SPN_CLUSTER_ID"]).spark_conf
```
Don't abuse named arguments!

Reply:
fixed